lib/apress/selenium_eti/spec/company_site/minieti_spec.rb

```diff
@@ -96,7 +96,7 @@
       before { @cs_eti_page.set_exists(CONFIG['eti']['exists']['in stock']) }
 
       it 'введенная цена отображается' do
-        expect(@cs_eti_page.exists_value).to include 'В наличии'
+        expect(@cs_eti_page.exists_value).to include /[Вв] наличии/
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the / if it should be a division.